apps/samenvvv/pages/index.tsx

```diff
@@ -16,62 +16,100 @@ import i18nConfig from '../next-i18next.config'
+            <Heading as="h3" size="xl" color="white">
+              {t('home.post-maker.title')}
+            </Heading>
+            <Text fontSize="xl" fontWeight="normal" maxWidth="2xl">
+              {t('home.post-maker.content')}
+            </Text>
+
+            <Button
+              as={Navigate}
+              href={link || '/'}
+              size={{ base: 'sm', md: 'lg' }}
+              fontWeight="semibold"
+              variant="solid"
+              colorScheme="primary.600"
+              bg="white"
+              color="primary.500"
+              boxShadow="lg"
+              _hover={{ color: 'white', bg: 'blackAlpha.100' }}
+            >
+              {t('home.post-maker.button')}
+            </Button>
+          </Stack>
+
+          {hashtag && (
             <Stack
-              direction={{ base: 'column', lg: 'row' }}
-              alignItems="center"
-              spacing={4}
+              direction={{ base: 'column', sm: 'row' }}
+              color="white"
+              spacing={6}
+              alignItems={'stretch'}
+              justifyContent={'space-between'}
             >
               <Stack
-                order={{ base: 2, lg: 1 }}
-                color="white"
-                spacing={8}
-                alignItems={{ base: 'center', lg: 'start' }}
-                flex={1}
-                textAlign={{ base: 'center', lg: 'left' }}
+                py={2}
+                justifyContent={'space-between'}
+                alignItems={{ base: 'center', sm: 'flex-start' }}
               >
-                <Heading as="h3" size="2xl" color="white">
-                  {t('home.post-maker.title')}
+                <Heading as="h3" size="xl" color="white">
+                  {hashtag.title}
                 </Heading>
-                <Text fontSize="xl" fontWeight="semibold">
-                  {t('home.post-maker.content')}
+                <Text fontSize="xl" fontWeight="normal" maxWidth="2xl">
+                  {hashtag.description}
                 </Text>
-
                 <Button
                   as={Navigate}
                   href={link || '/'}
-                  size="lg"
-                  variant="outline"
-                  colorScheme="primary"
-                  color="white"
+                  size={{ base: 'sm', md: 'lg' }}
+                  fontWeight="semibold"
+                  variant="solid"
+                  colorScheme="primary.600"
+                  bg="white"
+                  color="primary.500"
+                  boxShadow="lg"
                   _hover={{ color: 'white', bg: 'blackAlpha.100' }}
                 >
-                  {t('home.post-maker.button')}
+                  {t('read-more')}
                 </Button>
               </Stack>
-
-              <PostMakerIcon
-                order={{ base: 1, lg: 2 }}
-                boxSize={{ base: 300, lg: 500 }}
+              <WImage
+                width={480}
+                height={270}
```

Comment:
```suggestion
                ratio={16 / 9}
```

Using fixed width will break responsive design for small devices (~360px). By default we use [`fill`](https://nextjs.org/docs/api-reference/next/image#fill) property from Next.js which doesn't require width or height to be set.

Normally, people recommend using width and height for better performance. And we know dimensions of the images coming from our backend upfront. But this is something we should improve later in `WImage` component.